add a ref for inhouse consulting (#105)
DE-RSE/2023_paper-RSE-groups: paper.tex

There are at least three aspects to RSE pooling that research institutions can benefit from: funding, diverse knowledge, and support contacts.
The first, pooling of \textbf{funding}, allows organisations to invest in building up institutional knowledge by supporting RSEs to become experts.
A central RSE team on long-term contracts will act as a knowledge hub due to their accumulated experience in and support of several disciplines as well as established contacts within the organisation.

> This is comparable to commercial/industry R\&D departments or so-called inhouse consulting~\autocite{Grima_2011}, where key software architects and developers establish a knowledge hub and consult with as many projects as necessary.

Reviewer: ```suggestion
This is comparable to commercial/industry R\&D departments or so-called inhouse consulting~\autocite{Grima_2011}, where key software architects and developers establish a knowledge hub that can be consulted by project teams as necessary.
```
Presumably the internal consultants from the hub do not ask the projects for help.
